app/views/cars/show.haml

```diff
@@ -38,14 +38,14 @@
           %dd
             = category_name(@car.category)
           %dt{:style => "cursor: help;", :"data-toggle" => "tooltip", :"data-placement" => "top", :title => t("rva.cars.features.stock-tooltip.tooltip")}
-            = t("rva.cars.features.stock")
+            = t("rva.shared.stock.title")
           %dd
-            = "#{@car.stock? ? "Yes" : "No"}"
+            = "#{@car.stock? ? t("rva.shared.stock.true") : t("rva.shared.stock.false")}"
```

Comment:
Maybe even move these to misc string keys? Yes and No seem generic enough to me.

```suggestion
            = "#{@car.stock? ? t("rva.shared.stock.yes") : t("rva.shared.stock.no")}"
```